### PR TITLE
Project completed

### PR DESCRIPTION
**Pull request by alphayowakarindi**
At this juncture of development, we completed the project with all the required features.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+4. Run `ruby main.rb` to start the app
+
+### Run tests
+1. Run `gem install rspec` to install the `RSpec testing framework`
+2. Run `rspec spec` to test all the test files at once
+3. Run `rspec spec/name_of_test_file.rb` to test a particular file
+
+## Brief overview
+
+[Google Drive Link](https://drive.google.com/file/d/1Hq9qMQqJvaoQq7XeNukZkrBAut98VVlz/view?usp=share_link)
```
Comment (danielufeli) [inline README.md:29]: - [ ] Nice work done on your project and your video presentation was well explanatory, but as a requirement, the maximum length of the video is 5 minutes yours was 6.47 minutes, kindly reduce it.

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

SoCloseKeeganMichaelKeyGIF

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code✔️

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (alphayowakarindi) [reply README.md:29]: Hi @danielufeli, thank you so much for taking the time to leave this excellent review. Making the change and we really appreciate your input to make our code better. Cheers!

File: README.md [hunk 2]
```diff
@@ -1 +1,59 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+## 📝 License
+This project is MIT licensed.
+
+NOTE: we recommend using the MIT license - you can set it up quickly by using templates available on GitHub. You can also use any other license if you wish.
```
Comment (elmar8287) [inline README.md:59]: - [ ] OPTIONAL You have mentioned about MIT license, but there is no MIT.md file in your repo. Please, fix it before merging to the main branch 👍

**Review by elmar8287 — APPROVED** [review]
Hi, team.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- The video is fixed as required 👍  

### Optional suggestions

- Please, find my OPTIONAL suggestion in a comment under the review 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (alphayowakarindi) [reply README.md]: Hi @elmar8287, thank you so much for taking the time to leave this excellent review. We have made the change and we really appreciate your input to make our code better. Cheers!

Comment (elmar8287) [reply README.md]: @alphayowakarindi Happy coding! 🥇

**Pull request merged**